{"name": "测试数据：【 wintest1111，False，1004，查不到相关用户 】", "status": "broken", "statusDetails": {"message": "json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)", "trace": "self = <testcases.api_test.test_01_get_user_info.TestGetUserInfo object at 0x04F44050>, username = 'wintest1111', except_result = False, except_code = '1004', except_msg = '查不到相关用户'\n\n    @allure.story(\"用例--获取某个用户信息\")\n    @allure.description(\"该用例是针对获取单个用户信息接口的测试\")\n    @allure.issue(\"https://www.cnblogs.com/wintest\", name=\"点击，跳转到对应BUG的链接地址\")\n    @allure.testcase(\"https://www.cnblogs.com/wintest\", name=\"点击，跳转到对应用例的链接地址\")\n    @allure.title(\"测试数据：【 {username}，{except_result}，{except_code}，{except_msg} 】\")\n    @pytest.mark.single\n    @pytest.mark.parametrize(\"username, except_result, except_code, except_msg\",\n                             api_data[\"test_get_get_one_user_info\"])\n    def test_get_get_one_user_info(self, username, except_result, except_code, except_msg):\n        logger.info(\"*************** 开始执行用例 ***************\")\n        step_2(username)\n>       result = get_one_user_info(username)\n\ntestcases\\api_test\\test_01_get_user_info.py:54: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\ntestcases\\api_test\\user.py:33: in get_one_user_info\n    if res.json()[\"code\"] == 0:\nd:\\test\\flaskdemo-master\\venv\\lib\\site-packages\\requests\\models.py:900: in json\n    return complexjson.loads(self.text, **kwargs)\nC:\\Users\\videt\\AppData\\Local\\Programs\\Python\\Python36-32\\lib\\json\\__init__.py:354: in loads\n    return _default_decoder.decode(s)\nC:\\Users\\videt\\AppData\\Local\\Programs\\Python\\Python36-32\\lib\\json\\decoder.py:339: in decode\n    obj, end = self.raw_decode(s, idx=_w(s, 0).end())\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <json.decoder.JSONDecoder object at 0x03D542B0>, s = '<!doctype html><html lang=\"en\"><head><title>HTTP Status 400 – Bad Request</title><style type=\"text/css\">body {font-fa...ht:1px;background-color:#525D76;border:none;}</style></head><body><h1>HTTP Status 400 – Bad Request</h1></body></html>', idx = 0\n\n    def raw_decode(self, s, idx=0):\n        \"\"\"Decode a JSON document from ``s`` (a ``str`` beginning with\n        a JSON document) and return a 2-tuple of the Python\n        representation and the index in ``s`` where the document ended.\n    \n        This can be used to decode a JSON document from a string that may\n        have extraneous data at the end.\n    \n        \"\"\"\n        try:\n            obj, end = self.scan_once(s, idx)\n        except StopIteration as err:\n>           raise JSONDecodeError(\"Expecting value\", s, err.value) from None\nE           json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)\n\nC:\\Users\\videt\\AppData\\Local\\Programs\\Python\\Python36-32\\lib\\json\\decoder.py:357: JSONDecodeError"}, "description": "该用例是针对获取单个用户信息接口的测试", "steps": [{"name": "步骤1 ==>> 获取某个用户信息", "status": "passed", "parameters": [{"name": "username", "value": "'wintest1111'"}], "start": 1622569884841, "stop": 1622569884842}], "attachments": [{"name": "log", "source": "2809aa52-cba7-4a29-a4b2-ad5d3012b12d-attachment.txt", "type": "text/plain"}, {"name": "stderr", "source": "2441f9a9-5ddb-4379-a4d3-cd2d76b08100-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "username", "value": "'wintest1111'"}, {"name": "except_result", "value": "False"}, {"name": "except_code", "value": "'1004'"}, {"name": "except_msg", "value": "'查不到相关用户'"}], "start": 1622569884840, "stop": 1622569884858, "uuid": "30d046f6-8aa8-4e4d-b9d3-3dd806e66d56", "historyId": "cadac3d6893c9533e7359301271a9794", "testCaseId": "11ffae3263ffa61feddcb26de6c12bb7", "fullName": "testcases.api_test.test_01_get_user_info.TestGetUserInfo#test_get_get_one_user_info", "labels": [{"name": "epic", "value": "针对单个接口的测试"}, {"name": "severity", "value": "trivial"}, {"name": "story", "value": "用例--获取某个用户信息"}, {"name": "feature", "value": "获取用户信息模块"}, {"name": "tag", "value": "single"}, {"name": "parentSuite", "value": "testcases.api_test"}, {"name": "suite", "value": "test_01_get_user_info"}, {"name": "subSuite", "value": "TestGetUserInfo"}, {"name": "host", "value": "LAPTOP-1UC29CD2"}, {"name": "thread", "value": "21668-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testcases.api_test.test_01_get_user_info"}], "links": [{"type": "test_case", "url": "https://www.cnblogs.com/wintest", "name": "点击，跳转到对应用例的链接地址"}, {"type": "issue", "url": "https://www.cnblogs.com/wintest", "name": "点击，跳转到对应BUG的链接地址"}]}